Project/Source/Components/ComponentParticleSystem.cpp

```diff
@@ -421,6 +487,20 @@ void ComponentParticleSystem::InitParticleAnimationTexture(Particle* currentPart
 	}
 }
 
+void ComponentParticleSystem::InitParticleTrail(Particle* currentParticle) {
```

Comment:
Can you move this Init function and the one from above to the rest of Init functions? Thanks 🙏🏼